ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
+        # a new IdaDataFrame is created, rows will be selected only if all the listed columns have a NULL value.
+        """
+        
+        # Check if list columns is not empty
+        if len(columns)<1:
+            raise IdaDataFrameError("You must specify the columns as a list of eligible names")
+        # Check if column names are eligible
+        for column_name in columns:
+            if not isinstance(column_name, six.string_types):
+                raise TypeError("%s is not of string type"%(column_name))
+            if column_name not in self.columns:
+                raise ValueError("%s refers to a columns that doesn't exists in self"%(column_name))
+        
+        idadb = self._idadb
+        
+        if inplace == True:
+            # Ask for confirmation
+            print("The table %s will be physically modified." %self.tablename)
+            answer = input('Do you want to proceed? [y/n]')
+            if answer == 'n':
+                print('Then use the default option inplace = False')
+                return None
+            if answer == 'y':
+                print("Any IdaDataFrame pointing this table might be modified accordingly.")
+            else:
+                print("User input should be either 'y' or 'n'.")
+                return None
+            # Write DELETE query on original table
+            tablename = self.tablename
+            query = 'DELETE FROM %s WHERE "%s" IS NULL'%(tablename, columns[0])
+            
+            if len(columns)>1:
+                if logic == "any":
+                    for i in range(1,len(columns)):
+                        query = query + ' OR "%s" IS NULL'%columns[i]
+                if logic == "all":
+                    for i in range(1, len(columns)):
+                        query = query + ' AND "%s" IS NULL'%columns[i]                        
+            idadb.ida_query(query)
+
+        else:
+            
+            # SELECT statement on the original table to create a view
+            tablename0 = self.tablename 
+            query = 'SELECT * FROM %s WHERE "%s" IS NOT NULL'%(tablename0, columns[0])
+            
+            if len(columns)>1:
+                if logic == "any":
+                    for i in range(1,len(columns)):
+                        query = query + ' AND "%s" IS NOT NULL'%columns[i]
+                if logic == "all":
+                    for i in range(1, len(columns)):
+                        query = query + ' OR "%s" IS NOT NULL'%columns[i]                                   
+            
+            # Create view with this select statement
+            viewname = idadb._get_valid_tablename(prefix="VIEW_")
+            self._prepare_and_execute("CREATE VIEW " + viewname + " AS "+ query)
+
+            # Initiate the modified table under a random name
+            tablename = idadb._get_valid_tablename(prefix="DATA_FRAME_")
+            idadb._prepare_and_execute("CREATE TABLE %s LIKE %s"%(tablename,viewname))
```

Comment:
In https://github.com/ibmdbanalytics/ibmdbpy/pull/50/files#diff-51eaf414f17704158d33f72c3a0107f7L1877-L1878, you replaced create table like + insert into with create table as select. You can do so here as well.

Reply:
ok. Replaced by: `idadb._prepare_and_execute("CREATE TABLE %s AS (SELECT * FROM %s)"%(tablename,viewname))`